Cargo.toml

```diff
@@ -126,6 +126,7 @@ prost-types = "0.6.1"
 
 # GCP
 goauth = { version = "0.9.0", optional = true }
+gouth = { version = "0.1", optional = true }
```

Comment:
The reason I went for `gouth` is because I need to send (according to GCP docs) 3 scopes to be able to send metrics. `goauth` only supports sending one scope.